### PR TITLE
Updated script admin menu

### PR DESCRIPTION
**Pull request by tessawiedmann**
- Removed instructions column
- Added column names
- Added column for template levels
- Added column for contained levels
- Made menu on top of bubbles (z index was too low)

adminmenu1
image

### CODE REVIEW

**Comment by Hamms** [comment]
Can you confirm that the removed instructions aren't actually being used by anyone?

File: dashboard/app/views/scripts/show.html.haml [hunk 1]
```diff
@@ -70,11 +70,22 @@
...
           .span1
             = index + 1
           .span3
             = level.name
-          .span8
-            = level.instructions
+          .span3
+            = level.properties['project_template_level_name']
+          .span3
+            = level.properties['contained_level_names']
```
Comment (Hamms) [inline dashboard/app/views/scripts/show.html.haml]: nit; it'd look a lot nicer if you did something like `level.properties['contained_level_names'].join(", ")`

File: dashboard/app/views/scripts/show.html.haml [hunk 2]
```diff
@@ -70,11 +70,22 @@
...
       - levels = Script.get_from_cache(@script.name).stages.map{ |stage| stage.script_levels.map{ |sl| sl.level }}.flatten
+      .row
+        .span1
+          = "Index"
+        .span3
+          = "Level name"
+        .span3
+          = "Template level"
+        .span3
+          = "Contained levels"
```
Comment (Hamms) [inline dashboard/app/views/scripts/show.html.haml:81]: :+1:

**Comment by tessawiedmann** [comment]
@Hamms Confirmed by @mrjoshida that instructions aren't used in this menu

**Pull request merged**